external/kinesis-asl/src/main/python/examples/streaming/kinesis_wordcount_asl.py

```diff
@@ -26,6 +26,7 @@
     <stream-name> name of the Kinesis stream (ie. mySparkStream)
     <endpoint-url> endpoint of the Kinesis service
       (e.g. https://kinesis.us-east-1.amazonaws.com)
+    <region-name> region name of the Kinesis endpoint (e.g. us-east-1)
```

Comment:
BTW, did you run this Kinesis example during your verification on `us-east-1`?

Reply:
I ran it on `ap-northeast-1`.